src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -331,6 +337,10 @@ static boolean isIgnoreLockFile(Project project) {
         return project.hasProperty("ignoreLockFile");
     }
 
+    private static boolean isSkipWriteLocks(Project project) {
+        return project.hasProperty("gcvSkipWriteLocks");
```

Comment:
Happy to iterate on the prop name. Wanted to make it clear that this is specific for GCV and not other plugins that also use the `--write-locks` flag.